apps/ideaspace/src/components/modules/SubmissionForm/SubmissionForm.js

```diff
@@ -31,6 +31,18 @@ function SubmissionForm() {
     ['primary', 'close'],
   );
 
+  const getInvolveLevel = (involveString) => {
+    if (involveString.includes("and also believe")) {
+      return "highly";
+    } else if (involveString.includes("to help with workshopping")) {
+      return "medium";
+    }/* else if (involveString.includes("")) {
```

Comment:
Should we remove this?

Reply:
Leaving this, because this is not showing on the UI right now, but should be.